implementations/micrometer-registry-humio/src/main/java/io/micrometer/humio/HumioMeterRegistry.java

```diff
@@ -128,8 +128,7 @@ protected void publish() {
     }
 
     @Override
-    @NonNull
-    protected TimeUnit getBaseTimeUnit() {
+    protected @NonNull TimeUnit getBaseTimeUnit() {
```

Comment:
`@NonNull` is redundant here since the package-info is annotated with `@NullMarked`. I'll remove it.